pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -0,0 +1,28 @@
+[id="creating-a-product_{context}"]
+= Creating a product
+
+A product must exist in {ProductName} before you can assign a product name to the metadata of a module or assembly. If the product doesn't already exist, you need to create one.
+
+.Prerequisites
+
+* You are logged in.
+* You have Publisher permissions.
+
+.Procedure
+
+. Navigate to menu:Products[New Product].
+
+. Enter the following information:
+  Product Name:: This is the text that appears in {ProductShortName}, and the document preview, and in the user-facing content.
+  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+  Product Description (optional):: A description of the product.
+
+. Click btn:[Save].
+
+.Example
+
+Product Name:: .NET Core
+Product UrlFragment:: `net_core` 
+Product Description:: 
+
+After creating a product version `4.4-beta` and assigning a module the .NET Core product and version 4.4-beta, the resulting URL for that module is: `https://access.stage.redhat.com/documentation/en-us/topic/net_core/4.4-beta/_module-UUID_`
```

Comment:
I don't understand what information this sentence is trying to convey.

Reply:
@robin-owen Take a look now, I tried to clarify it.